pkg/rancher/migrations.go

```diff
@@ -90,10 +90,6 @@ func runMigrations(wranglerContext *wrangler.Context) error {
 		}
 	}
 
-	if err := migrateRKEClusterStates(wranglerContext); err != nil {
-		return err
-	}
-
 	return migrateImportedClusterFields(wranglerContext)
```

Comment:
I think we should remove the old configmaps, so maybe add a check on startup to do so.